go.mod

```diff
@@ -682,6 +682,10 @@ replace (
 
 // TODO(rfratto): remove forks when changes are merged upstream
 replace (
+	// TODO(tpaschalis) this is to remove global instantiation of plugins
+	// and allow non-singleton components.
+	github.com/google/cadvisor => github.com/grafana/cadvisor v0.0.0-20230927082732-0d72868a513e
```

Comment:
Do we have a link of the PR already? Would be good to add it here as part of the comment

Reply:
Yes, will do. If you want to take a look, the branch is [here](https://github.com/grafana/cadvisor/tree/grafana-v0.47-noglobals)